assets/prototype/domain/shared/predicates/datetimes/filters/activeOnly/index.test.ts

```diff
@@ -0,0 +1,23 @@
+import dates from '../tests/data';
+import activeOnly from './index';
+
+describe('activeOnly', () => {
+	it('Should return an array with active-only dates', () => {
+		const filteredDates = activeOnly(dates);
+
+		expect(Array.isArray(filteredDates)).toBe(true);
```

Comment:
Do we really need such checks when we have TS?

Reply:
Agree, I was a bit paranoic.

Reply:
It might be helpful to have a function emptyArray or similar for situations like these:
```
expect(Array.isArray(filteredDates)).toBe(true);
expect(filteredDates.length).toBe(0);
```

Reply:
Removed here: https://github.com/eventespresso/event-espresso-core/commit/af6fd58b92aff033ddb59b2fbc9b0dcdf19261fa